static/app/components/deprecatedforms/formField.tsx

```diff
@@ -120,13 +117,8 @@ export default class FormField<
     );
   };
 
-  getField() {
-    throw new Error('Must be implemented by child.');
-  }
-
-  getClassName(): string {
-    throw new Error('Must be implemented by child.');
-  }
+  abstract getField(): React.ReactNode;
+  abstract getClassName(): string;
```

Comment:
since we don't use abstract much - typescript will error if these aren't implemented by the parent class and we do not need the runtime check